src/client/Client.cc

```diff
@@ -14019,6 +14019,8 @@ int Client::_rename(Inode *fromdir, const char *fromname, Inode *todir, const ch
     else
       return -CEPHFS_EROFS;
   }
+
+  // don't allow cross-quota renames
```

Comment:
@vshankar 
Same for this PR too. These are the only changes, should this backport be tested and merged?

Reply:
https://github.com/ceph/ceph/pull/52579#pullrequestreview-1541632328

BTW, in the conflict section you mention about is_enabled() vs is_enable() and I understand that, but it fails to describe what the actual issue is. I guess in p/q release you have

src/mds/mdstypes.h::quota_info_t
```
  bool is_enable() const {
    return max_bytes || max_files;
  }
```

and in main/reef,

```
src/include/cephfs/types.h:  bool is_enabled(quota_max_t type=QUOTA_ANY) const
```

Since your fix in main/reef branch is logically similar to p/q releases, we could drop backport.

If that's the case, then this needs to be clearly explained in the backport tracker and the tracker can be closed.

Reply:
@rishabh-d-dave ping?

Reply:
@rishabh-d-dave ping?